### DISCUSSION TITLE
Using custom reference annotation

### DISCUSSION THREAD

**Discussion by ashwinikumarkulkarni** [stuart-lab/signac#1068] [Q&A] [answered]
Hello,
I have used a UCSC genome and Gencode annotation for snRNA-seq alignment and counting. How can I create or use same genome and annotation for ATAC-seq? Thanks!
-Ashwinikumar

**Comment by timoast** [comment, marked as answer]
Please see https://satijalab.org/signac/articles/faq.html#an-annotation-or-genome-sequence-for-my-organism-is-not-available-on-bioconductor-what-do-i-do
